components/formats-gpl/src/loci/formats/in/NativeND2Reader.java

```diff
@@ -2235,9 +2233,12 @@ else if (key.startsWith("Line:")) {
           if (value.endsWith("Active")) {
             int first = key.lastIndexOf(":") + 1;
             int last = key.lastIndexOf(";");
+            if(last-first < 0){
```

Comment:
But you could add one of the whitespaces after the `if` here :smile: (i.e. `if (last...`)